client/components/videos-ui/modal-footer-bar.jsx

```diff
@@ -1,24 +1,49 @@
-import { Gridicon } from '@automattic/components';
+import { Button, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 
 import './modal-footer-bar.scss';
 
-const ModalFooterBar = ( { onBackClick = () => {}, course = {} } ) => {
+const ModalFooterBar = ( { onBackClick = () => {}, course = {}, isCourseComplete = false } ) => {
 	const translate = useTranslate();
 	const onBackLinkClick = ( event ) => {
 		recordTracksEvent( 'calypso_courses_mobile_back_click', {
 			course: course?.slug,
 		} );
 		onBackClick( event );
 	};
+	const onStartWritingClick = () => {
+		recordTracksEvent( 'calypso_courses_cta_click', {
+			course: course?.slug,
+		} );
+	};
 
 	return (
-		<div className={ 'videos-ui__bar videos-ui__modal-footer-bar' }>
-			<a href="/" onClick={ onBackLinkClick }>
-				<Gridicon icon="chevron-left" size={ 24 } />
-				<span>{ translate( 'Back' ) }</span>
-			</a>
+		<div className="videos-ui__footer-bar">
+			<div
+				className={ classNames( 'videos-ui__bar videos-ui__modal-footer-bar', {
+					'videos-ui__course-completed': isCourseComplete,
+				} ) }
+			>
+				<a href="/" onClick={ onBackLinkClick }>
+					<Gridicon icon="chevron-left" size={ 24 } />
+					<span>{ translate( 'Back' ) }</span>
+				</a>
+				{ isCourseComplete && (
+					<p>
+						{ translate(
+							"You did it! Now it's time to put your skills to work and draft your first post."
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **16** times:
`translate( 'You did it! Now it's time to put your skill to work and draft your first post.' )` *ES Score: 9*